src/include/daos_pool.h

```diff
@@ -190,7 +190,7 @@ struct daos_pool_cont_info {
  *			-DER_NONEXIST	Pool is nonexistent
  */
 int
-daos_pool_connect(const char *pool, const char *sys, unsigned int flags,
+daos_pool_connect2(const char *pool, const char *sys, unsigned int flags,
 		  daos_handle_t *poh, daos_pool_info_t *info, daos_event_t *ev);
```

Comment:
I think that we want to keep advertising the daos_pool_connect symbol here and redirect it to daos_pool_connect2 with a #define.

Reply:
We do have "#define daos_pool_connect daos_pool_connect2" at the end of src/include/daos_pool.h (this file). We still use "daos_pool_connect" in other c code.

Reply:
Hmm, a bit confused here, if daos_pool_connect2 will not be exported anyway. why not remove daos_pool_connect2 and  just use daos_pool_connect() here?

Reply:
I just re-arranged the macro of daos_pool_connect. Please check out commit 5. Thank you!

Reply:
After merging master branch, CI issue is gone.